src/main/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -379,6 +379,14 @@ class AvailableItem {
 
 enum AvailableItemStatus {
     AVAILABLE, PICKED, RECALLED, HOLD, NOT_AVAILABLE
+
+    static getList() {
```

Comment:
Follow the existing convention for listing enum values and rename the second method to be more readable.
```
static list() { ... }
static listUnavailable() { ... }
```